xbmc/win32/Win32DelayedDllLoad.cpp

```diff
@@ -70,5 +70,5 @@ FARPROC WINAPI delayHookFailureFunc (unsigned dliNotify, PDelayLoadInfo pdli)
 }
 
 // assign hook functions
-PfnDliHook __pfnDliNotifyHook2 = delayHookNotifyFunc;
-PfnDliHook __pfnDliFailureHook2 = delayHookFailureFunc;
```

Comment:
title isn't really the issue, disabling code and then merging without pinging a platform maintainer is the failure here.
If it fails to compile then PR it and ping someone that can help out, might take a day or two more depending on availability..
Looking into it now so no reason to revert